Sources/NeedleFoundationTest/MockComponentPathBuilder.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+
+import Foundation
+import NeedleFoundation
+
+/**
+ This function allows you to get access to a mock component path builder that you can use to
+ build mock component paths of type MockComponentPath. You can use these paths to register mock
+ dependency providers during testing.
+ */
+public func mockComponentPathBuilder() -> MockComponentPathBuilder {
```

Comment:
Can we have properly formatted comment please? `///` and `- returns:`.

Reply:
Updated.